main.css

```
@@ -30,9 +31,7 @@ nav .wrapper{
 }
 nav .login-button{
```

nav에서 사용하는 부분에 #gnb를 넣어주면 사이드 이펙트가 줄 수 있을 것 같아요!
```suggestion
nav#gnb .login-button{
```